Makefile

```diff
@@ -66,13 +66,17 @@ start-mongo:
      --rm \
      --detach \
      --name skynet-accounts-mongo-test-db \
-     -p 127.0.0.1:17017:27017 \
+     -p 17017:17017 \
      -e MONGO_INITDB_ROOT_USERNAME=admin \
      -e MONGO_INITDB_ROOT_PASSWORD=aO4tV5tC1oU3oQ7u \
-     mongo
+     -v $(shell pwd)/test/fixtures/mongo_keyfile:/data/mgkey \
+	mongo:4.4.1 mongod --port=17017 --replSet=skynet --keyFile=/data/mgkey
+	sleep 3 # wait for mongo to start before we try to configure it
+	# Initialise a single node replica set.
+	docker exec skynet-accounts-mongo-test-db mongo -u admin -p aO4tV5tC1oU3oQ7u --port 17017 --eval "rs.initiate({_id: \"skynet\", members: [{ _id: 0, host: \"localhost:17017\" }]})"
```

Comment:
Yes. This thing is spinning up a test container that's exposing its port on the host system. If the host system is running another mongo container (as mine is 90% of the time) this will fail for a silly and mundane reason. So, I switched it to a custom port in order to avoid that.